examples/nodejs/plt/v1/modify-list.ts

```diff
@@ -125,6 +127,30 @@ const client = new ConcordiumGRPCNodeClient(
             const result = await client.waitForTransactionFinalization(transaction);
             console.log('Transaction finalized:', result);
 
+            if (result.summary.type !== TransactionSummaryType.AccountTransaction) {
+                throw new Error('Unexpected transaction type: ' + result.summary.type);
+            }
+
+            switch (result.summary.transactionType) {
+                case TransactionKindString.TokenGovernance:
+                    result.summary.events.forEach((e) => {
+                        if (e.event.tag !== TokenEventType.Module) {
+                            throw new Error('Unexpected event type: ' + e.event.tag);
+                        }
+                        console.log('Token module event:', e.event, Cbor.decode(e.event.details));
+                    });
+                    break;
+                case TransactionKindString.Failed:
+                    if (result.summary.rejectReason.tag !== RejectReasonTag.TokenGovernanceTransactionFailed) {
+                        throw new Error('Unexpected reject reason tag: ' + result.summary.rejectReason.tag);
+                    }
+                    const details = Cbor.decode(result.summary.rejectReason.contents.details);
+                    console.error(result.summary.rejectReason.contents, details);
```

Comment:
Somtimes you use `console.error` and sometimes you throw an error.  Just wonder if you want to align the flow.